src/bmeg/util/cli.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" utility, sets common arguments   """
+import argparse
+import logging
+
+
+def default_argument_parser():  # pragma: no cover
```

Comment:
I'm nitpicking, but I don't like how often these types of "pragma" statements are showing up in the code. Instead of caring about an absolute code coverage metric, we could instead pay attention to the relative change over time.

Reply:
per discussion - limited to command line outside of normally testable code